hardhat.config.ts

```diff
@@ -23,22 +26,17 @@ const config: HardhatUserConfig = {
     ]
   },
   networks: {
-    // main: {
-    //   chainId: 137,
-    //   url: `${process.env.MAIN_RPC_URL}`,
-    //   accounts: [`0x${process.env.MAIN_PRIVATE_KEY}`],
-    //   gasPrice: 200000000000
-    // },
-    // mumbai: {
-    //   chainId: 80001,
-    //   url: `${process.env.MUMBAI_RPC_URL}`,
-    //   accounts: [`0x${process.env.MUMBAI_PRIVATE_KEY}`]
-    // },
-    // amoy: {
-    //   chainId: 80002,
-    //   url: `${process.env.AMOY_RPC_URL}`,
-    //   accounts: [`0x${process.env.AMOY_PRIVATE_KEY}`]
-    // },
+    main: {
```

Comment:
you should comment this, otherwise  it will be error during deploy script run even if I don't use main at all.

Reply:
Fixed